docs/product-assembly_case-management-getting-started/src/main/asciidoc/case-management-designing-a-case-proc.adoc

```diff
@@ -1,2 +1,99 @@
-[id='']
-= TODO
\ No newline at end of file
+[id='case-management-designing-a-case-proc']
+= Designing a case
+
+Cases are designed using the *Case definition* design editor in {CENTRAL}. The case design is the basis of case management and sets out the specific goals and tasks for each case. The design can be added to and modified dynamically during runtime.
+
+To demonstrate how to design a case, {PRODUCT} includes the IT Orders sample project in {CENTRAL}, which includes the following case definition:
+
+image::itorders-orderhardware-process.png[IT Orders case design]
+
+You can refer to and interact with this case definition by importing the sample project in {CENTRAL}. 
+
+.Prerequisite 
+* You have followed the procedure in <<case-management-creating-a-case-proc>> to create a new case in {CENTRAL}.
+
+.Procedure
+. *Create a new case definition.*
++
+In the *Assets* window, click *Add Asset* in the upper-right corner of the project perspective and select *Case definition*. 
+
+. Complete the required information in the *Create new Case definition* window:
```

Comment:
Here and elsewhere, for the bulleted parameters below, I'd go with colon instead of dash, per consistency with other docs, and cap the text following, per IBM SG. Example, "`name`: The name of the case..."